### ISSUE TITLE
Doesn' find the module torch.backends

### ISSUE THREAD

**Issue by twoertwein** [microsoft/pyright#2232]
**Describe the bug**
The pytorch module `torch.backends` is not detected by pyright.

File structure:
```
torch/__init__.py
torch/backends/__init__.py
```

**To Reproduce**
```py
import torch
torch.backends
```
error: "backends" is not a known member of module (reportGeneralTypeIssues)
